ansible/roles/load-balancer/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
IMO we should remove this role all-together so people steer towards Octavia by default